distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Select2,Apache Software License 2.0.txt

```diff
@@ -0,0 +1 @@
+Select2,GNU General Public License v2.0 only.txt
```

Comment:
The symbolic link here doesn't reflect any decision on which license Keycloak chooses to use for Select2. That project doesn't offer discrete license texts in its source code, so both license files have identical text, so they get symlinked for deduplication purposes.